src/normalized.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. While you have added Arial and Helvetica as fallbacks, it's also a good practice to include a generic family name such as 'sans-serif'.